addons/web/controllers/webmanifest.py

```diff
@@ -45,7 +45,7 @@ def webmanifest(self):
         Using this metadata, user agents can provide developers with means to create user
         experiences that are more comparable to that of a native application.
         """
-        web_app_name = request.env['ir.config_parameter'].sudo().get_param('web.web_app_name', 'Odoo')
+        web_app_name = self._get_app_name()
```

Comment:
extract to sw

Reply:
Instead of this, I chose the other option which we discussed: fetch the manifest from the service (I no longer need http service, but only import the get method). It is only fetched once by the service, when accessing the appName.

It seemed IMO too complicated for something as simple to try to register into the service worker currently. If we need it elsewhere later, we can still refactor it at that moment